src/lib/core/style/focus-classes.ts

```diff
@@ -3,7 +3,10 @@ import {Observable} from 'rxjs/Observable';
 import {Subject} from 'rxjs/Subject';
 
 
-export type FocusOrigin = 'mouse' | 'keyboard' | 'program';
+const TOUCH_BUFFER_MS = 650;
```

Comment:
Yeah I think the timeout is still fine. I took it from https://github.com/angular/material/pull/7965#r59114101